File: app/src/main/java/com/zetapush/testsdk/NoTokenStorage.java
package com.zetapush.testsdk;

/**
 * Created by damien on 20/07/17.
 */

public class NoTokenStorage implements StorageTokenInterface {

    @Override
    public void saveToken(String token) {
        // Do nothing
    }

    @Override
    public String getToken() {
        return null;
    }

    @Override
    public void clearToken() {
        // Do nothing
    }
}

File: android-library/src/main/java/com/zetapush/library/NoTokenStorage.kt
package com.zetapush.library

/**
 * Created by damien on 20/07/17.
 */

class NoTokenStorage : StorageTokenInterface {

    override fun saveToken(token: String?) {
        // Do nothing
    }

    override val token: String?
        get() = null

    override fun clearToken() {
        // Do nothing
    }
}
